databricks_cli/dbfs/api.py

```diff
@@ -113,16 +113,25 @@ def get_status(self, dbfs_path, headers=None):
         json = self.client.get_status(dbfs_path.absolute_path, headers=headers)
         return FileInfo.from_json(json)
 
+    # Single variation of put implemented.
+    # See https://docs.databricks.com/dev-tools/api/latest/dbfs.html#put
+    # @put_file() is for multipart file upload.
```

Comment:
This comment is a bit confusing for me. What do you mean by `Single variation`? Why do you say `@put_file() is for multipart file upload`? Did you mean `self.client.put` instead?

Reply:
I tried to mean the following method. I will make it more clear and also mention the underlying `put` method used.